### DISCUSSION TITLE
progress bar

### DISCUSSION THREAD

**Discussion by deguangchow** [microsoft/vcpkg#19067] [Ideas]
I hope you can add a progress bar when downloading or compiling, just like 'apt' in Linux.

**Comment by cbezault** [comment]
I'm not sure this is going to happen any time soon but it's a reasonable feature request, maybe a community member can pick it up.

**Comment by NikBomb** [comment]
I would like to contribute, I use vcpkg and I feel that a progress bar is a very good feature to add.

**Comment by PhoebeHui** [comment]
The feature would be added by PR https://github.com/microsoft/vcpkg/pull/12550

**Comment by dannypike** [comment]
Even adding the package number to the report would help. I tried adding a [new issue](https://github.com/microsoft/vcpkg/issues/20973) to suggest that idea but it was closed and transferred to this one.

Which is a pity because it means that they will probably not do it, given that this is now a discussion and has been open for so long.

**Comment by dannypike** [comment]
I wrote a small utility to help me with this. It is targetted at Windows, is simplistic, but it covers the one case that I want `vcpkg` to handle:

[https://github.com/dannypike/vcpkg-tee](https://github.com/dannypike/vcpkg-tee)
